vendor-onboarding-checklist.md

```diff
@@ -0,0 +1,175 @@
+# Vendor onboarding checklist
+
+## Purpose
+
+This document supports the Division of Public Assistance Eligibility
+Information System Replacement (EIS-R) Project. Specifically, this
+document addresses our agile acquisition practice and onboarding new
+vendor staff. Within agile acquisition, we anticipate bringing in a
+potentially different vendor every three to four months, possibly more
+often when we begin running multiple acquisitions simultaneously. With
+short timelines, this onboarding checklist will bring new vendor staff
+up to speed as quickly as possible.
+
+# Audience
+
+This document intends to inform EIS-R State of Alaska project staff and
+the vendors we engage with in various product-increment acquisitions.
+
+# Roles
+
+This section identifies the anticipated vendor roles that will be
+engaged with the
+project.
+
+| **Role**                 | **Description**                                                                             |
+| ------------------------ | ------------------------------------------------------------------------------------------- |
+| Project manager          | Manage the project                                                                          |

```

Comment:
@nfaulknerAK Is the project manager and the scrum master two different vendor resources in your mind?  It seems a little redundant to me.

Reply:
I don't think they are two different resources. I agree that would be redundant.

Reply:
Per slack convo, combining PM and Scrum master roles in vendor-onboarding-checklist.